### PR TITLE
Qna fixes

### PR DESCRIPTION
**Pull request by UgurGumushan**
Achieves: 
Screenshot 2024-02-21 at 12 07 05

### CODE REVIEW

File: src/routes/backoffice.ts [hunk 1]
```diff
@@ -305,7 +306,10 @@ export const updatePersonHandler = async (req, res) => {
   if (questionSet) {
     await triggerWebhook({
       type: PersonWebhookEvent.QUESTIONS_REQUIRE_RESPONSE,
-      payload: questionSet,
+      payload: {
+        question_set_id: questionSet.id,
```
Comment (lera) [inline src/routes/backoffice.ts:310]: thank you!

File: src/routes/backoffice.ts [hunk 2]
```diff
@@ -282,10 +282,11 @@ export const updatePersonHandler = async (req, res) => {
...
   let questionSet = null;
-  const shouldGenerateQuestionSet = [
-    req.body.customer_vetting_status,
-    req.body.risk_classification_status,
-  ].includes(CustomerVettingStatus.INFORMATION_REQUESTED);
+  const shouldGenerateQuestionSet =
+    req.body.customerVettingStatus ===
+      CustomerVettingStatus.INFORMATION_REQUESTED ||
+    req.body.riskClassificationStatus ===
+      RiskClarificationStatus.INFORMATION_REQUESTED;
```
Comment (lera) [inline src/routes/backoffice.ts:289]: why was it changed?

File: src/routes/backoffice.ts (hunk 1 above)
Comment (UgurGumushan) [inline src/routes/backoffice.ts:310]: https://docs.solarisgroup.com/api-reference/onboarding/webhooks/#tag/Webhook-events/paths/questions_require_response/post

File: src/routes/backoffice.ts (hunk 2 above)
Comment (UgurGumushan) [reply src/routes/backoffice.ts:289]: request body has props in camel case, at least when set from panel. I'll check again if we set them differently through api

Comment (UgurGumushan) [reply src/routes/backoffice.ts:289]: Reference: 
https://github.com/kontist/mock-solaris/blob/f41cf09b50ea36a3a7abec510473712f236c864c/src/routes/backoffice.ts#L268

**Review by p-janik — APPROVED** [review]
Thanks a lot!

**Comment by lera** [comment]
but here were failing tests. please fix them
Screenshot 2024-02-21 at 11 58 07

**Pull request merged**